CMakeLists.txt

```
@@ -63,12 +63,9 @@ set_directory_properties(PROPERTIES
     CXX_VISIBILITY_PRESET hidden
 )
 
-option(BUILD_WITH_QT6 "Build Quotient with Qt 6 (EXPERIMENTAL)" OFF)
-if (BUILD_WITH_QT6)
-    set(QUOTIENT_LIB_NAME ${PROJECT_NAME}Qt6)
-else()
-    set(QUOTIENT_LIB_NAME ${PROJECT_NAME})
-endif()
+set(QtMinVersion "6.0")
```

While we're at it, let's bump the Qt version:
```suggestion
set(QtMinVersion "6.4")
```
All recent stable distros have it already, and 6.4 has some really nice additions like `StringLiterals`.

---

I ran too fast, after all :( Ubuntu 24.04 is still not even released, let alone used in a GitHub runner. I'll do this in a separate PR then.